.tool/lint

```diff
@@ -10,6 +10,7 @@ for d in $(find . -type d -not -iwholename '*.git*' -a -not -iname '.tool' -a -n
 		--exclude='.*_test\.go:.*error return value not checked.*\(errcheck\)$' \
 		--exclude='duplicate of.*_test.go.*\(dupl\)$' \
 		--exclude='cmd\/client\/.*\.go.*\(dupl\)$' \
+		--exclude='server\/seccomp\/.*\.go.*$' \
```

Comment:
It contains C code (from the libseccomp package) and gometalinter is not understanding it yet.